### ISSUE TITLE
Why use CLI when there’s a configuration file?

### ISSUE THREAD

**Issue by swashcap** [MRN-Code/penny-collector#16]

**Comment by swashcap** [comment]
Perhaps part of a different discussion, but I’m not sure we want to mutate configuration like this: https://github.com/MRN-Code/penny-collector/blob/5ec7c0c0ee03f7c07f8d5863200f86cb5eac183d/src/utils/build-parse-cli.js#L73

**Comment by cdaringe** [comment]
I agree, it was the wrong pattern.  My bad.  globals => app, and cli input
piecewise update app/global/whatever you want to call em.  #hackathon

**Comment by swashcap** [comment]
@cdaringe Totally. #hackathonLyfe
